aws.env

```diff
@@ -0,0 +1,7 @@
+NODE_ENV="aws"
+
+# Google Geocode API
+GOOGLE_MAPS_KEY='AIzaSyBj1aLo0X1uH7CKoi3bXPFPdX6eeCra4e8'
+
+# Debug
+LOG_LEVEL='error'
```

Comment:
Help me understand why you added just `error` logs to the AWS lambda in production. What is your opinion about `INFO`, `WARN`, `DEBUG`, and TRACE logs in production?

Reply:
Well there was no specific reason to set level to 'error' in aws env file. I was just testing all the levels with cloudwatch and ended up on error. To answer your question that what should we be doing in production then under normal circumstances production log level should be 'Info'. As far as the different log levels are concerned:

**Info**: These logs gives useful information about the system in general. This is sort of information which one don't really care about under normal circumstances but very helpful in crisis situation.
**WARN**: Gives you info about anything that has a potential to cause error OR fatal exception. Usually we have mechanisms for auto-recovery(or system gets recovered on its own sometimes) with problems logged on this level but this keeps us alerted that what can go wrong in future.
**Debug/Trace**: I would like to address debug and trace in same part. There is always a debate between level of these log levels in heirarchy but in my experience, debug < trace. This is the level which we only turn on in the situation of diagnoses. If we consider debug < trace side, then debug logs fine grained information(but less granular than Trace) about operation being performed while trace level gives us the most fine grained info(even step by step sometimes) about the operation. We only enable it to diagnose the things in production because it has a performance impact on system.